lib/utils/index.js

```diff
@@ -533,6 +533,29 @@ class Utils {
   static shouldReplaceStack(err) {
     return !alwaysDisplayError(err);
   }
+
+  static findTSConfigFile() {
+    const projectTsFileLocation1 = path.join(process.cwd(), 'nightwatch', 'tsconfig.json');
+    const projectTsFileLocation2 = path.join(process.cwd(), 'tsconfig.nightwatch.json');
+    if (Utils.fileExistsSync(projectTsFileLocation1)) {
+      return projectTsFileLocation1;
+    } else if (Utils.fileExistsSync(projectTsFileLocation2)) {
+      return projectTsFileLocation2;
+    }
+
+    return '';
+  }
+
+  static loadTSNode(projectTsFile) {
+    try {
+      require('ts-node').register({
+        esm: false,
+        project: projectTsFile
+      });
+    } catch (err) {
+      Logger.error(err);
```

Comment:
here we can check if `ts-node` has been installed in the current project and if not, display the appropriate error message.